apidoc/Facebook.yml

```diff
@@ -522,6 +522,23 @@ methods:
         Set the [appid](Modules.Facebook.appid) property as well as the `permissions` property
         before calling `authorize()`.
 
+  - name: setLoginBehavior
```

Comment:
Because it's a property, name it "loginBehavior". The setter and getter are created automatically.